alert-policies/apache-traffic-server/CPUPercent.yml

```diff
@@ -0,0 +1,40 @@
+# Name of the alert
+name: CPU Percent
+
+# Description and details
+description: |+
+  This alert is triggered if the CPU usage exceeds 95% for 5 minutes.
```

Comment:
```suggestion
  This alert is triggered if the CPU usage exceeds 90% for 5 minutes.
```

Reply:
Hi @pkosanam ,
As per your suggestion, I have made changes. Kindly review it.

Thanks...